tests/ui/feature-gates/feature-gate-unsafe_fields.rs

```diff
@@ -0,0 +1,26 @@
+//@ compile-flags: --crate-type=lib
+//@ revisions: with_gate without_gate
+//@ [with_gate] check-pass
+
+#![allow(incomplete_features)]
```

Comment:
```suggestion
```
Testing that the `incomplete_features` lint is emitted is important, since experimental features must trigger the lint until their RFC is accepted.